app/src/main/java/org/oppia/android/app/topic/TopicFragmentPresenter.kt

```diff
@@ -52,7 +54,7 @@ class TopicFragmentPresenter @Inject constructor(
     )
     binding.lifecycleOwner = fragment
     this.storyId = storyId
-    viewPager = binding.root.findViewById(R.id.topic_tabs_viewpager) as ViewPager
+    viewPager2 = binding.root.findViewById(R.id.topic_tabs_viewpager) as ViewPager2
```

Comment:
Can we just keep the variable as 'viewPager'? The type is confusing as '2', but the variable makes it seem like there are two view pagers.

Reply:
done